packages/lib/fs-driver-base.ts

```diff
@@ -143,7 +143,7 @@ export default class FsDriverBase {
 	// or assign to option using .bind(fsDriver())
 	public async cacheCssToFile(cssStrings: string[]) {
 		const cssString = Array.isArray(cssStrings) ? cssStrings.join('\n') : cssStrings;
-		const cssFilePath = `${Setting.value('tempDir')}/${md5(escape(cssString))}.css`;
+		const cssFilePath = `${Setting.value('tempDir')}/${md5(cssString)}.css`;
```

Comment:
I didn't check the whole PR yet but leaving a note here just in case I forget: we need escaping here due to a bug in the md5 package: https://github.com/pvorb/node-md5/issues/41 Was there any reason for removing this call? (Performance I guess? but in this case we can't remove this)

Reply:
Sorry, I didn't consider the requirement such as emoji. The reason of removing is only for performance. 

To cover all possible `string` values for [pvorb/node-md5](https://github.com/pvorb/node-md5),  `md5(Buffer.from('...'))` is a faster alternative of `md5(escape('...')` in node.js. It bypasses pvorb/node-md5's slow string-to-bytes conversion. If md5 is 2.3.0+, TextEncoder is also a good alternative.

Anyway, I'll revert it.

Reply:
As a result of my review, escape() is replaced by Buffer.from() in 828963a.

Buffer.from() covers all possible values of string, and `md5(Buffer.from(...))` is much faster than bare `md5(...)` and `md5(escape(...))`.